README.md

```diff
@@ -3,123 +3,56 @@
 
 [![Powered by NumFOCUS](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](https://numfocus.org)
 
-JuMP is a domain-specific modeling language for **[mathematical optimization]**
-embedded in **[Julia]**. It currently supports a number of open-source and
-commercial solvers ([Artelys Knitro], [BARON], [Bonmin], [Cbc], [CDCS], [CDD],
-[Clp], [COSMO], [Couenne], [CPLEX], [CSDP], [ECOS], [FICO Xpress], [GLPK],
-[Gurobi], [Ipopt], [Juniper], [MOSEK], [NLopt], [OSQP], [ProxSDP], [SCIP],
-[SCS], [SDPA], [SDPT3], [SeDuMi], [Tulip]) for a variety of problem classes, including
-**[linear programming]**, **[(mixed) integer programming]**,
```

Comment:
If we're deleting these keywords from the README, we should add some as GitHub package topics for better searchability.

Reply:
Added keywords as Github package topics.

Reply:
![image](https://user-images.githubusercontent.com/8177701/102135533-00e9de80-3ebd-11eb-83dd-8e360f7c0735.png)